apps/web/server/routers/viewer/eventTypes.tsx

```diff
@@ -18,6 +18,22 @@ function isPeriodType(keyInput: string): keyInput is PeriodType {
   return Object.keys(PeriodType).includes(keyInput);
 }
 
+/**
+ * Ensures that it is a valid HTTP URL
+ * It automatically avoids
+ * -  XSS attempts through javascript:alert('hi')
+ * - mailto: links
+ */
+function assertValidUrl(url) {
```

Comment:
```suggestion
function assertValidUrl(url: string) {
```

We're enabling strict mode soon so should avoid untyped parameters.